modules/raft/src/main/java/org/apache/ignite/raft/jraft/disruptor/StripedDisruptor.java

```diff
@@ -149,12 +152,12 @@ public StripedDisruptor(
             String stripeName = format("{}_stripe_{}", poolName, i);
 
             Disruptor<T> disruptor = DisruptorBuilder.<T>newInstance()
-                .setRingBufferSize(bufferSize)
-                .setEventFactory(eventFactory)
-                .setThreadFactory(threadFactorySupplier.apply(nodeName, stripeName))
-                .setProducerType(ProducerType.MULTI)
-                .setWaitStrategy(useYieldStrategy ? new YieldingWaitStrategy() : new BlockingWaitStrategy())
-                .build();
+                    .setRingBufferSize(bufferSize)
```

Comment:
Formatting seems incorrect

Reply:
I applied fromat that we use for all cod in our project, except the ignite-raft.
So you think this is because our format never applied to the code before, but it is right.